src/display/editor/editor.js

```diff
@@ -494,6 +536,29 @@ class AnnotationEditor {
   get contentDiv() {
     return this.div;
   }
+
+  /**
+   * If true then the editor is currently edited.
+   * @return {boolean}
```

Comment:
Looking at our other getters, elsewhere in the code-base, I believe that you want the following for things to work generally:
```suggestion
   * @type {boolean}
```